### ISSUE TITLE
test.py tests what?

### ISSUE THREAD

**Issue by kajalsanklecha** [iMoonLab/MeshNet#21]
Thank you for the code!

But had a doubt. Your Readme.md file says that the code implements both classification and retrieval. Can you please specify what does test.py exactly do?

**Comment by thss15fyt** [comment]
Thanks for your interests.

The `test.py` loads pre-trained model, and evaluates its performance of classification and retrieval on the validation set.
